### PR TITLE
Expose main_module & main_module_path to api

### PR DESCRIPTION
**Pull request by rcourt**

### CODE REVIEW

File: src/zelos/api/zelos_api.py [hunk 1]
```diff
@@ -632,6 +633,29 @@ def thread(self) -> Thread:
...
+        """
+        Returns the parsed main module, if it exists, otherwise returns None.
+
+        :type: :py:class:`zelos.plugin.Parser`
+
+        """
+        return self.internal_engine.main_module
+
+    @property
+    def main_module_path(self) -> Optional[str]:
```
Comment (kvalakuzhyzp) [inline src/zelos/api/zelos_api.py:647]: main_binary_path

File: src/zelos/api/zelos_api.py [hunk 2]
```diff
@@ -632,6 +633,29 @@ def thread(self) -> Thread:
         """
         return self.internal_engine.current_process.current_thread
 
+    @property
+    def main_module(self) -> Optional[Parser]:
```
Comment (kvalakuzhyzp) [inline src/zelos/api/zelos_api.py:637]: api name -> main_binary
Parser -> ParsedBinary

Comment (rcourt) [inline src/zelos/api/zelos_api.py:637]: main_*binary*

File: src/zelos/api/zelos_api.py (hunk 1 above)
Comment (rcourt) [inline src/zelos/api/zelos_api.py:647]: main_*binary*_path

File: src/zelos/api/zelos_api.py [hunk 3]
```diff
@@ -632,6 +633,29 @@ def thread(self) -> Thread:
...
+        """
+        Returns the parsed main module, if it exists, otherwise returns None.
+
+        :type: :py:class:`zelos.plugin.Parser`
```
Comment (rcourt) [inline src/zelos/api/zelos_api.py:641]: Update name of `Parser` class to be `ParsedBinary` for clarity

**Pull request merged**